docs/docs/metrics.md

```diff
@@ -117,24 +94,23 @@ Please note that prior to <span class="label label-default">v0.15</span> these w
 number of HTTP requests received by Marathon is available under `.count`.
 There are more metrics around HTTP requests under the
 `org.eclipse.jetty.servlet.ServletContextHandler` prefix.
-For more infos, look at
+For more information, look at
 [the code](https://github.com/dropwizard/metrics/blob/796663609f310888240cc8afb58f75396f8391d2/metrics-jetty9/src/main/java/io/dropwizard/metrics/jetty9/InstrumentedHandler.java#L41-L42).
 
 ### JVM
 
-`jvm.threads.count` (meter) - the total number of threads. If this is above >500, this
-is generally a bad sign.
+`jvm.threads.count` (meter) - The total number of threads. If this is above >500, this
+is generally a bad sign. <-- Uh, can we be more precise here? -->
```

Comment:
There are too many options to say something about the root cause. This sentence should say: the number should be below 500.